plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorSmokeTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.BaseConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.testng.annotations.BeforeClass;
+
+import java.util.regex.Pattern;
+
+import static io.trino.plugin.bigquery.BigQueryQueryRunner.BigQuerySqlExecutor;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestBigQueryConnectorSmokeTest
+        extends BaseConnectorSmokeTest
```

Comment:
This connector sometimes reaches API limit in CI. How about extending `AbstractTestQueryFramework` instead and adding minimum tests described in  https://github.com/trinodb/trino/pull/14729/files#r1009087323 if the test purpose is verifying only `bigquery.metadata-cache-ttl` config property?

Reply:
The idea with smoke test was to make sure metadata caching doesn't break any functionality. But now I see that none of the JDBC connectors with metadata caching have a test like that.

I like your suggestion @ebyhr.